wordle-web/src/App.vue

```diff
@@ -1,28 +1,97 @@
 <template>
-  <header>
-    <div class="wrapper">
-      <nav>
-        <RouterLink to="/">Home</RouterLink> | <RouterLink to="/wordle">Wordle</RouterLink> |
-        <RouterLink to="/about">About</RouterLink>
-      </nav>
-      <v-btn @click="setInverseTheme"> Inverse Theme </v-btn>
-      <v-btn @click="setDarkTheme"> Dark Theme </v-btn>
-    </div>
-  </header>
-
-  <RouterView />
+  <div :class="themeClass">
+    <v-card class="mx-auto mb-12" max-width="448">
+      <v-layout>
+        <v-app-bar class="custom-app-bar" color="white" density="compact">
+          <template v-slot:prepend>
+            <RouterLink to="/">
+              <Icon icon="tabler:go-game" color="#155" width="50" height="50" />
+            </RouterLink>
+          </template>
+
+          <RouterLink to="/">
+            <h1 class="titleClass">Wordle</h1>
```

Comment:
So in the requirements for the assignment is says that the title should be on the left side of the app bar with the icon. I didn't mark you down for it since I think having it in the middle of the app bar looks good too. But I felt it is worth mentioning.

Reply:
Ty.